tools/go-cli/go-whisk/whisk/shared.go

```diff
@@ -20,7 +20,7 @@ import "encoding/json"
 
 type KeyValue struct {
     Key   string        `json:"key,omitempty"`
-    Value interface{}   `json:"value"`     // Whisk permits empty values, do don't omitemtpy
+    Value interface{}   `json:"value"`     // Whisk permits empty values, do don't omitempty
```

Comment:
@dubeejw @mdeuser is this supposed to be `don't omit empty`? the commit doesn't make any sense.